third_party/bazel/blade_helper/build_defs.bzl.tpl

```diff
@@ -1,2 +1,17 @@
 def python_bin_path():
     return "%{PYTHON_BIN_PATH}"
+
+def if_tensorrt_enabled(x):
+    if %{TENSORRT_ENABLED}:
+        return select({"//conditions:default": x})
+    return select({"//conditions:default": []})
+
+def if_internal(x):
+    if %{IF_INTERNAL}:
+        return select({"//conditions:default": x})
+    return select({"//conditions:default": []})
+
+def if_hie_enabled(x):
+    if %{IF_HIE}:
```

Comment:
Is it possible to hide all hie-related options?

Reply:
If we have a pybind.h for internal pybind wrapper, this is possible. Try fixing now..